docs/tutorial/workarounds.md

```diff
@@ -0,0 +1,142 @@
+# Common tricks used in (T)FHE
+
+As explained in the [Basics of FHE](../getting-started/fhe_basics.md), the challenge for developers
+is to adapt their code to fit FHE constraints. In this document we have collected some common examples
+to illustrate the kind of optimization one can do to get better performance.
+
+{% hint style="info" %}
+All code snippets provided here are for educational purpose. In future version of Concrete, some
+functions described here could be directly available in a more generic and efficient form.
+These code snippets are coming from support answers in our [community forum](https://community.zama.ai)
+{% endhint %}
+
+
+## Minimum for Two values
+
+In this first example, we compute a minimum by creating a difference between the two numbers $$y$$ and $$x$$
+and conditionally remove this diff from $$y$$ to either get $$x$$ if $$y>x$$ or $$y$$ if $$x>y$$:
```

Comment:
in the .md, in GT, it's spread horribly, because of bad management of double dollar
I would recommend just to say: we compute the minimum of x and y, and remove the formula.

Reply:
it's working well on Gitbook, you could see the preview

Reply:
yes sure, but it's terrible in GitHub. Don't you want a simple way with " we compute the minimum of x and y", which would look nice on GitHub as well?

Reply:
yes if you want